styles/styles.css

```
@@ -0,0 +1,217 @@
+@font-face {
+    font-family:"Rubik" ;
+    font-style: normal;
+    font-weight:400 ;
+    src: url("../fonts/rubik/rubik-400.woff2") format("woff2");
+    font-display: swap;
+}
+@font-face {
+    font-family:"Rubik" ;
+    font-style: normal;
+    font-weight:700 ;
+    src: url("../fonts/rubik/rubik-700.woff2") format("woff2");
+    font-display: swap;
+}
+@font-face {
+    font-family:"Raleway" ;
+    font-style: normal;
+    font-weight:800 ;
+    src: url("../fonts/raleway/raleway-800.woff2") format("woff2");
+    font-display: swap;
+}
+body {
+    font-family: "Rubik";
+    font-style: normal;
+    font-size:18px ;
+    line-height:30px ;
+    background-color:#F0F0F0;
+    color: black;
+  }
+  .page-header {
+    background-color: #ffe17f;
+    color: black;
+  }
+  .main-index {
+    color:black;
+    background-color:white;
+  }
+  .navigation-link {
+      color: black;
+      text-decoration: none;
+      align-items: center;
+  }
+  .navigation-navigation-link {
+      font-family: "Raleway";
+      color: black;
+      line-height:21px ;
+      font-weight: 800px;
+      text-align: center;
+      text-transform: uppercase;
+      text-decoration: none;
+
+  }
+  .title {
+      font-family: "Raleway";
+      font-size: 50px;
+      font-weight: 800;
+      line-height: 50px;
+      color: black;
+      
+  }
+  .description
+  {
+      color: #444444;
+  }
+ .button {
+    font-family: "Raleway";
+    font-size: 18px;
+    line-height: 21px;
+    font-weight: 800;
+    text-align: center;
+    color: black;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+  .indicators {
+    font-size: 36px;
+    line-height: 30px;
+  }
+  .function {
+    font-size: 16px;
+    line-height: 30px;
+    color: #444444;
+    
+  }
+  
+.navigation-image {
+background-image: url("../images/menu/yellow.svg");
+background-size: 160px 160px;
+color: black;
+          
+  }
```

```suggestion
.navigation-image {
  background-image: url("../images/menu/yellow.svg");
  background-size: 160px 160px;
  color: black;
}
```
сделай красивый формат (нужно исправить везде)